app/models.py

```diff
@@ -43,6 +43,22 @@ def insert_roles():
     def __repr__(self):
         return '<Role \'%s\'>' % self.name
 
+class IdlingIncident(db.Model):
+    __tablename__ = "incidents"
+    vehicle_id = db.Column(db.Integer, primary_key=True)
+    location = db.Column(db.String(64))
```

Comment:
I would move location into its own data type. It would be somewhat like [this](https://github.com/hack4impact/gpcah/blob/master/app/models.py#L80) except that we only need `latitude`, `longitude` and `address`. `address` is just the string that the user submitted (which we attempt to geocode).